### PR TITLE
Continuing type refactor

### PR DESCRIPTION
**Pull request by michelchan**

### CODE REVIEW

File: morphir-ir/shared/src/main/scala/zio/morphir/ir/ValueModule.scala [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package zio.morphir.ir
 
 import zio.morphir.ir.{Literal => Lit}
-import zio.morphir.ir.TypeModule.Type
+import zio.morphir.ir.Type.Type.{unit => UnitType}
```
Comment (DamianReeves) [inline morphir-ir/shared/src/main/scala/zio/morphir/ir/ValueModule.scala:4]: This should be `Unit`, no?

File: morphir-ir/shared/src/test/scala/zio/morphir/ir/ValueModuleSpec.scala [hunk 2]
```diff
@@ -7,14 +7,14 @@ import zio.morphir.testing.MorphirBaseSpec
 import zio.morphir.syntax.ValueSyntax
 import ValueModule.Value
 import ValueModule.ValueCase._
-import zio.morphir.ir.TypeModule.Type
+import zio.morphir.ir.Type.{Type => irType}
```
Comment (DamianReeves) [inline morphir-ir/shared/src/test/scala/zio/morphir/ir/ValueModuleSpec.scala:10]: Please keep aliases for types uppercase.

File: morphir-ir/shared/src/main/scala/zio/morphir/ir/ValueModule.scala (hunk 1 above)
Comment (michelchan) [reply morphir-ir/shared/src/main/scala/zio/morphir/ir/ValueModule.scala:4]: the original is unit. If you're referring to the alias I gave it, I wanted to differentiate it from the Value unit

**Review by DamianReeves — APPROVED** [review]
LGTM

**Pull request merged**